api/tinygo/runnable/internal/ffi/ffi.go

```diff
@@ -32,3 +32,10 @@ func result(size int32) ([]byte, runnable.HostErr) {
 
 	return result, nil
 }
+
+func addVar(name, value string) {
+	namePtr, nameSize := rawSlicePointer([]byte(name))
+	valuePtr, valueSize := rawSlicePointer([]byte(value))
+
+	C.add_ffi_var(namePtr, nameSize, valuePtr, valueSize, Ident())
```

Comment:
I see the `Ident()` returns the global variable `ident_`, but that's not set anywhere, not exported, so by default it's ... 0? Does that get set during build with an -X flag?

This is not blocking, curious how that works / why this works. 🙂

Reply:
Nice, thank you! TIL